### ISSUE TITLE
what is the best solution to indentity diffirent id type in value ?

### ISSUE THREAD

**Issue by leizard** [Yermo/rich_textarea#8]
When I got the value like this 
>[o=pirate3] [o=tag2]
How can I tell which id belongs to user, which belongs to tag.
thank you

**Comment by Yermo** [comment]
Good point. It'd a bad example. Add something to your value field so you can identify which is which. I usually have the value use an array format such as {type:'person',value:'pirate3'} instead of just 'pirate3'.

**Comment by leizard** [comment]
yeah I think I tried that solution

`{ label: labe;, value: { value: 'pirate1', content: '<div>content<div>', type: 'user' } }`

but when I get date-type of element it return undefined. this is what I tried to change in the jquery.rich_textarea.js (line 5261)

` text_string += "[o=" + $( elem ).attr( 'data-value' ) + $( elem ).attr( 'data-type' )   "]";`

Maybe I did it wrong ?

**Comment by Yermo** [comment]
For example try changing the demo, instead of changing the source itself:

{ label: 'tag1', value: { value: {type: 'tag',value:'tag1'}, content: '<span class="ui-button ui-state-default ui-widget ui-corner-all ui-button-text-only">Tag1</span>' } },

**Comment by leizard** [comment]
ok I will try it. thank you :D

**Comment by leizard** [comment]
ok now I struggle to get data value with this object

{ label: 'tag1', value: { value: {type: 'tag',value:'tag1'}, content: 'Tag1' } }

the $( elem ).attr( 'data-value' ) return a string "[Object Object]".
